README.md

```diff
@@ -57,49 +67,28 @@ try {
 }
 ```
 
-
 Calling `getAccount` with the existing account index, will retrieve the account stored on the device.
+
 ```java
 if (kinClient.hasAccount()) {
     account = kinClient.getAccount(0);
```

Comment:
Why do we need an index here if we support multiple accounts with the storeKey parameter?

Reply:
history, that come first (multiple accounts), let's discuss merging this 2 capabilities (multiple account and key based store)